styleguide/source/_patterns/00-atoms/04-images/03-logo-reversed.twig

```diff
@@ -0,0 +1 @@
+<a href="{{ link.pages-homepage }}"><img src="../../assets/images/logo-reversed.svg" class="logo" alt="American Medical Association" /></a>
```

Comment:
This is rendering out as `<a href="0">`. Please set a default to link to `https://www.ama-assn.org`. I realize this prob came from the other logo, which also seems broken.